UPDATING.md

```diff
@@ -40,6 +40,193 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Move GCP operators and hook to core
```

Comment:
```suggestion
### Move GCP operators and hooks to core
```
But i think, it should be more generic. This should take into account our future plans for changing operator names.

Reply:
What about:
"Changes to import paths and names of GCP operators and hooks"

Reply:
LGTM